src/stylesheets/variables.less

```diff
@@ -59,6 +59,12 @@
 @impac-dashboard-source-color:                  @impac-dashboard-title-label-color;
 @impac-dashboard-buttons-border-radius:         4px;
 
+//Info-text
+@impac-dashboard-info-text-color:               white;
+@impac-dashboard-info-text-background-color:    rgb(23,38,45);
```

Comment:
@alexnoox @claire00 maybe you could use `rgb(23,38,45, 0.7)` so then the opacity can be customised with a variable, and there is no need to use the `opacity: 0.7;` rule in the `dashboard.less` file.